lib/FocusTrap.js

```diff
@@ -1,31 +1,91 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import ReactDOM from 'react-dom';
+import {addFocusListener, addBlurListener} from './focusListeners';
 
 class FocusTrap extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.focusListener;
+    this.blurListener;
+    this.willWrap = !this.props.noWrapper;
+    this.setupNativeListeners = this.setupNativeListeners.bind(this);
+    this.removeNativeListeners = this.removeNativeListeners.bind(this);
+  }
+
   static propTypes = {
     onFocus: PropTypes.func,
     onBlur: PropTypes.func,
     component: PropTypes.any,
+    noWrapper: PropTypes.bool,
```

Comment:
I am not a huge fan of this property name. Working with negatives can be harder and confusing. It is also a little obtuse - something like `wrapChildren` or `useFocusWrapper` is more direct and explanatory about the behaviour the user can expect. 

This will of course require complementing the logic throughout the component, and setting a default value of `true`.

Reply:
I understand the opposition to 'turning on an off switch' with the negatory prop name. Will work with updating it to use `useFocusWrapper`.